docs/designers-developers/developers/tutorials/javascript/extending-the-block-editor.md

```diff
@@ -34,15 +34,17 @@ See [Packages](/docs/designers-developers/developers/packages.md) for list of av
 
 After you have updated both JavaScript and PHP files, go to the block editor and create a new post.
 
-Add a quote block, and in the right sidebar under Styles, you will see your new Fancy Quote style listed.  Click the Fancy Quote to select and apply that style to your quote block:
+Add a quote block, and in the right sidebar under Styles, you will see your new Fancy Quote style listed.  
+
+Click the Fancy Quote to select and apply that style to your quote block
```

Comment:
It seems the sentence "Click the Fancy Quote to select and apply that style to your quote block" now does not contains a punctuation mark, should we keep the two dots or add a normal dot?